localgov_demo.module

```diff
@@ -100,13 +121,5 @@ function localgov_demo_modules_installed($modules, $is_syncing) {
       }
     }
 
-    // Regenerate all path aliases.
-    // This needs to happen after entity reference fixes.
-    $nids = \Drupal::entityTypeManager()->getStorage('node')->getQuery()->execute();
-    $nodes = \Drupal::entityTypeManager()->getStorage('node')->loadMultiple($nids);
-    foreach ($nodes as $node) {
-      \Drupal::service('pathauto.generator')->updateEntityAlias($node, 'update');
-    }
```

Comment:
Any reason why this is has been removed?

Reply:
Intersting... that looks like a mistake. Thanks!

Reply:
Thanks Stevo, fixed this with https://github.com/localgovdrupal/localgov_demo/pull/59/commits/a4e34f200a33c53ea482c74623bb84a1d5d69ccb